index.js

```diff
@@ -500,45 +517,24 @@ RingPop.prototype.lookupN = function lookupN(key, n) {
 };
 
 RingPop.prototype.reload = function reload(file, callback) {
```

Comment:
I think the right thing to do here is to add a `// Deprecated` comment to the `/admin/reload` handler and add a warning log when that endpoint is called. Later we can remove it.